metaworld/envs/mujoco/sawyer_xyz/v2/sawyer_basketball_v2.py

```diff
@@ -143,7 +141,18 @@ def compute_reward(self, action, obs):
             sigmoid='long_tail',
         )
 
-        object_grasped = reward_utils.gripper_caging_reward(self, action, obj)
+        obj_radius = 0.025
+        object_reach_radius=0.01
+        pad_success_margin = 0.06
+        x_z_margin = 0.005
+
```

Comment:
Why do you define these constants here? If you're going to use keyword args to the caging function anyway, it's already clear what the numbers represent. No need to define local variables that only get used once. This applies to the other env diffs as well